### DISCUSSION TITLE
Possible MQTT (mqtt_as) usage without Wifi

### DISCUSSION THREAD

**Discussion by nastiliano** [micropython/micropython#13298] [General]
Hello,

It's possible to ignore config["ssid"] and config["wifi_pw"] due to previously self-configured WLAN settings?

Thank you very much.

Happy new year.

**Comment by prabhu-yu** [comment]
I wanted to use mqtt_as with PPP-over-serial.
ie, I don't want to use WiFi.
Is that possible?
